app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt

```diff
@@ -391,13 +394,35 @@ class ImageRecsFragment : SuggestedEditsItemFragment(), ImageRecsDialog.Callback
 
         binding.publishProgressBar.postDelayed({
             if (isAdded) {
+                showConfetti(false)
                 binding.publishOverlayContainer.visibility = GONE
                 callback().nextPage(this)
                 callback().logSuccess()
             }
         }, duration * durationBoost)
     }
 
+    private fun showConfetti(shouldShowConfetti: Boolean) {
+        binding.successConfettiImage.visibility = if (shouldShowConfetti) VISIBLE else GONE
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            // Change statusBar abd actionBar color
+            requireActivity().window.statusBarColor = if (shouldShowConfetti) ResourceUtil.getThemedColor(requireContext(),
+                    R.attr.image_recs_confetti_background_color) else Color.TRANSPARENT
+            (requireActivity() as AppCompatActivity).supportActionBar!!.setBackgroundDrawable(if (shouldShowConfetti)
+                ColorDrawable(ResourceUtil.getThemedColor(requireContext(), R.attr.image_recs_confetti_background_color)) else null)
```

Comment:
Looks like the navigation bar color is not correct. Should it be the same color as the progress bar view background instead?

Reply:
I too thought so... but Robin okayed it this morning... so just want to see what he thinks... or do you feel I should just change to progress bar color?